### PR TITLE
version 2.0.0

### PR DESCRIPTION
**Pull request by aboedo**
- removes deprecated `makePurchase`, replaced by `purchaseProduct`
 - iOS: 
     - added new method, `syncPurchases`, that enables syncing the purchases in the local receipt with the backend without risking a password prompt. The method was already available on Android.
     - added a new method, `presentCodeRedemptionSheet`, for offer codes redemption.
- Bumped `purchases-hybrid-common` to 1.5.0 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/1.5.0)
- Bumped `purchases-ios` to 3.9.2 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/3.9.2)
- Bumped `purchases-android` to 4.0.1 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.0.1)

### CODE REVIEW

File: src/ios/CDVPurchasesPlugin.m [hunk 1]
```diff
@@ -93,73 +86,61 @@ - (void)purchasePackage:(CDVInvokedUrlCommand *)command
...
-{
+- (void)createAlias:(CDVInvokedUrlCommand *)command {
     NSString *newAppUserID = [command argumentAtIndex:0];
 
     [RCCommonFunctionality createAlias:newAppUserID completionBlock:[self getResponseCompletionBlock:command]];
 }
 
-- (void)identify:(CDVInvokedUrlCommand *)command
-{
+- (void)identify:(CDVInvokedUrlCommand *)command {
```
Comment (aboedo) [inline src/ios/CDVPurchasesPlugin.m:108]: formatting here was inconsistent, so I just moved the file to this format, which we use in the native sdk

**Comment by aboedo** [comment]
@vegaro any thoughts on the failing integration tests? I'm not sure what's going on

**Comment by vegaro** [comment]
Looking @aboedo

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,9 +26,9 @@
     - Added support for OneSignal via `setOnesignalId`
     - Added `setMediaSource`, `setCampaign`, `setAdGroup`, `setAd`, `setKeyword`, `setCreative`, and `collectDeviceIdentifiers`
          https://github.com/RevenueCat/purchases-android/pull/184
-- Bumped `purchases-hybrid-common` to 1.4.3 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases)
-- Bumped `purchases-ios` to 3.7.2 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases)
-- Bumped `purchases-android` to 3.5.0 [Changelog here](https://github.com/RevenueCat/purchases-android/releases)
+- Bumped `purchases-hybrid-common` to 1.4.3 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/1.4.3)
+- Bumped `purchases-ios` to 3.7.2 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/3.7.2)
+- Bumped `purchases-android` to 3.5.0 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/3.5.0)
```
Comment (vegaro) [inline CHANGELOG.md:31]: Thanks for fixing these!

**Review by vegaro — APPROVED** [review]

**Pull request merged**